sympy/utilities/iterables.py

```diff
@@ -819,6 +819,42 @@ def sift(seq, keyfunc, binary=False):
     return T, F
 
 
+def tied_max(iterable, key):
+    """
+    Return the list of items from *iterable* which returns
+    highest value when *key* is applied.
+
+    Examples
+    ========
+
+    >>> from sympy.utilities.iterables import tied_max
+    >>> iterable = ['abc', 'de', 'fgh', 'i']
+    >>> key = lambda x: len(x)
+    >>> tied_max(iterable, key)
+    ['abc', 'fgh']
+
+    See Also
+    ========
+    core.operations.AssocOpDispatcher
+
```

Comment:
```suggestion
```
I'd probably just remove this, "see also" is usually for functions which are similar, not callers. The rest of the docstring looks good, thanks!

Reply:
Thanks for the suggestion.